server/src/config.mjs

```diff
@@ -53,6 +56,45 @@ const config = {
 // Layer in configuration specified via .mf.config.json file
 config.dotConfig = dotConfig;
 
+/* 
+* @function:compareConfigs
+* @Description: To compare changed and original config files and perform dynamic config updates
+* @param {Object} changedConfig - config that contains updates/additions wrt original config objects
+* @param {Object} loadedConfig - original config
+*/
+function compareConfigs(changedConfig, loadedConfig) {
+  // Iterate through new and changed config objects in changedConfig
+  for (const obj of changedConfig) {
+    if (obj.type == 'new') {
+      const newNameToCheck = obj.name;
+      const readme = obj.readme;
+      if (newNameToCheck && !searchJSONforParam(loadedConfig, newNameToCheck)) {
```

Comment:
JSON is a string serialization format... so technically, I think you're really 'just' searching an object for a param. I don't think it's critical, but you could/eventually should rename this function something like 'searchObjectForParam' or perhaps better 'searchObjectForKey'. Minor, but sometimes naming subtleties can help/hurt.